### ISSUE TITLE
Bundled Minicron run CLI behaving differently

### ISSUE THREAD

**Issue by robinjoseph08** [jamesrwhite/minicron#155]
I downloaded the ZIP of v0.8.0 and was trying to get it set up, but I noticed that all of my `minicron run` commands were failing:

```bash
$ minicron -v
minicron 0.8.0

$ minicron run 'echo hello' --dry-run
error: A valid command to run is required! See `minicron help run`.
```

I dug a bit further and found out the reason why was because `args` isn't being interpreted as `['echo hello']` as expected, it's being interpreted as `['echo', 'hello']` instead and that causes the exception to be raised here: https://github.com/jamesrwhite/minicron/blob/v0.8.0/lib/minicron/cli/commands.rb#L121-L123.

Interestingly enough though, when I cloned this repo, set it up, and ran `./bin/minicron run 'echo hello' --dry-run`, it actually worked perfectly fine.

I'm not sure sure if this is a deployment/binary issue, but it looks like the source is correct.

**Comment by robinjoseph08** [comment]
Another difference between the binary minicron and the source is that `minicron server start` doesn't automatically daemonize in the bundled version.

**Comment by jamesrwhite** [comment]
Yeah there a few issues with the `0.8.0` release unfortunately, the daemonizing issue I've resolved already. I'll look into the other issue though, I'm not sure why that would be specific to the binary version.

I'm planning to get a `0.8.1` release out this weekend so I'll try and include a fix for this in there too.

**Comment by jamesrwhite** [comment]
I believe https://github.com/jamesrwhite/minicron/releases/tag/v0.8.1 fixes this so I'm going to close, let me know if that's not the case.

**Comment by andofrjando** [comment]
It seems that this is not fixed as of 0.8.1 (or 0.8.3).

When I use the zipped bundle:
```
$ ./minicron -v
minicron 0.8.1

$ ./minicron run 'echo hello' --dry-run --trace
A valid command to run is required! See `minicron help run`
```
I noticed that the ```--trace``` option doesn't actually provide any back trace in this case.

Running a command with no options doesn't fail:
```
$ ./minicron run 'date' --dry-run
Sun Nov  8 23:09:09 AWST 2015
```

One interesting behaviour I noticed while troubleshooting:
```
$ date -v                           # 'date' obviously doesn't have a -v switch
date: invalid option -- 'v'
Try 'date --help' for more information.
```
But try this:
```
$ ./minicron run 'date -v' --dry-run
minicron 0.8.1
```
There's still issues parsing the command, despite being quoted. As I mentioned before, this is still the case with the packaged zip version 0.8.3. I'm trying all this on Ubuntu 14.04.3

I'm didn't spend too much time figuring out how to run from the cloned repo, so I gave up and was unable to test with a git source version.

**Comment by jamesrwhite** [comment]
I see the issue now, the [wrapper script](https://github.com/jamesrwhite/minicron-build/blob/master/build/packaging/wrappers/minicron.sh#L18) is removing the quotes. So instead of `run 'date -v' --dry-run` minicron is being passed `run date -v --dry-run` which explains this odd behaviour.

**Comment by jamesrwhite** [comment]
I've uploaded patched [`0.8.3`](https://github.com/jamesrwhite/minicron/releases/tag/v0.8.3) zip files, the only difference being the change above to the wrapper script. I've tested with all the above examples and it does appear to be behaving correctly, on the OSX release anyway, I don't see any reason why the linux builds would be different.

As a side note the `--trace` option is indeed rather pointless at the moment, it's added to exception messages by the underlying CLI library I'm using. I'm hoping to swap that out for another one soon.

Thanks for all the help with this.
